### ISSUE TITLE
vc4_vec broken on 6.7

### ISSUE THREAD

**Issue by 6by9** [raspberrypi/linux#5894]
### Describe the bug

Observing that the VEC fails to enable modes on 6.7 (probably 6.6 as well).
vc4_vec_encoder_atomic_check is failing at https://github.com/raspberrypi/linux/blob/rpi-6.6.y/drivers/gpu/drm/vc4/vc4_vec.c#L760
```
[    6.654936] vc4_vec_encoder_atomic_check: mode "720x480i": 60 13500 720 736 800 858 480 486 492 525 0x48 0x1a
[    6.655043] vc4_vec_encoder_atomic_check: mode->crtc_(timings) 720 736 800 858 240 243 246 262
```

This check was added via #4406 and is only a downstream patch. 

The drm_mode for the analog interlaced modes are now coming from the core framework (`fill_analog_mode` via `drm_analog_tv_mode`) vs being baked into vc4, and the timing has changed slightly
Was `13.500 720/14/64/60/-     480/7/6/32/-`
Now `13.500 720/16/64/58/-     480/6/6/33/-`
Horizontal timing change to the porches will be a slight horizontal shift.
Vertical timing changes may imply the PV is wanting a slight munge so that it give the VEC the right signals at the desired time.

### Steps to reproduce the behaviour

Enable `dtoverlay=vc4-kms-v3d,composite`, reboot, and observe the framebuffer not come up, and `kmstest` can't enable the default modes (encoder atomic_check fails).

### Device (s)

Raspberry Pi CM4

### System

6.7.1

**Comment by 6by9** [comment]
Simplest solution is to revert https://github.com/raspberrypi/linux/commit/2ef74e9687c92e26d4b047c2499c96b215ec7f00 (on 6.6).

**Comment by 6by9** [comment]
Fixed via #5898
